pyoozie/coordinator.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+from __future__ import unicode_literals
+
+from datetime import timedelta
+from enum import Enum
+
+from pyoozie.tags import Xml, Parameters, validate
+
+
+class ExecutionOrder(Enum):
+    """Execution order used for coordinator jobs."""
+
+    FIFO = 'FIFO'
+
+    LIFO = 'LIFO'
+
+    LAST_ONLY = 'LAST_ONLY'
+    # "When LAST_ONLY is set, an action that is WAITING or READY will be SKIPPED when the current time is past the
+    # next action's nominal time.  For example, suppose action 1 and 2 are both WAITING , the current time is
+    # 5:00pm, and action 2's nominal time is 5:10pm. In 10 minutes from now, at 5:10pm, action 1 will become
+    # SKIPPED, assuming it doesn't transition to SUBMITTED (or a terminal state) before then. Another way of
+    # thinking about this is to view it as similar to setting the timeout equal to the frequency, except that the
+    # SKIPPED status doesn't cause the coordinator job to eventually become DONEWITHERROR and can actually become
+    # SUCCEEDED (i.e. it's a "good" version of TIMEDOUT ). LAST_ONLY is useful if you want a recurring job, but do
+    # not actually care about the individual instances and just always want the latest action."
```

Comment:
Do we really need Oozie doc pasted here? (I know, was there in the original source too.)